new_testing_2/features.feature

```diff
@@ -0,0 +1,57 @@
+
+
+# features for automation assingment
+
+Feature: Passing parameter to steps
+
+  Scenario: check the login page is opened
+
+    Given I have to open "http://www.phptravels.net/"
```

Comment:
`Given I have to open "http://www.phptravels.net/"` as `Given open "http://www.phptravels.net/"`